### PR TITLE
chore: Stops using Network org in tests

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Stops using Network org in tests.

This will simplify testing infrastructure and will allow an easier reuse of resources.

Link to any related issue(s): CLOUDP-236296

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code
- [ ] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [ ] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: .github/workflows/acceptance-tests-runner.yml [hunk 1]
```diff
@@ -532,15 +521,17 @@ jobs:
         run: bash ./scripts/generate-credentials-with-sts-assume-role.sh
       - name: Acceptance Tests
         env:
+          MONGODB_ATLAS_BASE_URL: ${{ inputs.mongodb_atlas_base_url }}
+          MONGODB_ATLAS_ORG_ID: ${{ inputs.mongodb_atlas_org_id }}
+          MONGODB_ATLAS_PUBLIC_KEY: ""
+          MONGODB_ATLAS_PRIVATE_KEY: ""
```
Comment (AgustinBettati) [inline .github/workflows/acceptance-tests-runner.yml:524]: I assume this is to ensure no public/private key are provided in the test group right?

**Review by EspenAlbert — APPROVED** [review]
LGTM
I like that it removes the dependency on an existing PROJECT_ID and nice cleanup of using constants 👍

**Review by oarbusi — APPROVED** [review]

File: .github/workflows/acceptance-tests.yml [hunk 2]
```diff
@@ -62,8 +60,6 @@ jobs:
       test_group: ${{ inputs.test_group }}
       aws_region_federation: ${{ vars.AWS_REGION_FEDERATION }}
       mongodb_atlas_org_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_ORG_ID_CLOUD_QA || vars.MONGODB_ATLAS_ORG_ID_CLOUD_DEV }}
-      mongodb_atlas_org_id_network: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_ORG_ID_CLOUD_QA || vars.MONGODB_ATLAS_ORG_ID_CLOUD_DEV_NETWORK }}
-      mongodb_atlas_project_id_network: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_PROJECT_ID_CLOUD_QA || vars.MONGODB_ATLAS_PROJECT_ID_CLOUD_DEV_NETWORK }}
```
Comment (AgustinBettati) [inline .github/workflows/acceptance-tests.yml:66]: just to be sure I understand, with the adjustments in this PR we no longer depend on any existing project right?

**Review by AgustinBettati — APPROVED** [review]
LGTM, leaving 2 doubts from my side

File: .github/workflows/acceptance-tests-runner.yml (hunk 1 above)
Comment (lantoli) [reply .github/workflows/acceptance-tests-runner.yml:524]: correct

File: .github/workflows/acceptance-tests.yml (hunk 2 above)
Comment (lantoli) [reply .github/workflows/acceptance-tests.yml:66]: correct

**Pull request merged**